### PR TITLE
Typescript guide to get started

### PR DESCRIPTION
**Pull request by yairopro**

### CODE REVIEW

File: packages/docs/src/introduction.stories.mdx [hunk 1]
```diff
@@ -42,10 +42,43 @@ to setup a simple, web-only React app with built-in support for aliasing
...
+npm install --save react-native-web
+npm start
+```
+
+**With typescript:**
+```
+npx create-react-app my-app --template typescript
+cd my-app
 npm install react-native-web
+npm install --save-dev @types/react-native
```
Comment (necolas) [inline packages/docs/src/introduction.stories.mdx]: Are these types correct for react-native-web though?

Comment (yairopro) [reply packages/docs/src/introduction.stories.mdx]: RN-web has the same signatures as RN, doesn't it ?
I might be wrong. But which component or module would have a different interface ?

Comment (necolas) [reply packages/docs/src/introduction.stories.mdx]: No, the types are not exactly the same, e.g., for styles. This is true of every fork of RN

Comment (yairopro) [reply packages/docs/src/introduction.stories.mdx]: Ah ok

**Pull request closed without merging**